include/bout/field_visitor.hxx

```diff
@@ -22,7 +22,7 @@ public:
   virtual void accept(Field3D &f) = 0;
   virtual void accept(Vector2D &f) = 0;
   virtual void accept(Vector3D &f) = 0;
-  
+  virtual ~FieldVisitor() = default;
 };
```

Comment:
semicolon?

Reply:
Required here :)

Reply:
I am so confused...

Reply:
It's the end of a `class` declaration

Reply:
but the end of a template class doesn't need one?